src/app.js

```diff
@@ -1 +1,27 @@
 'use strict';
+
+const { generateNumber } = require('./numGenerator');
+const { bullsAndCows } = require('./bullsAndCows');
+const { inputFunc } = require('./inputOutput');
+
+const secretNumber = generateNumber();
+
+function game() {
+  inputFunc('Enter a 4-digit number: ', (guess) => {
+    if (!isValidGuess(guess)) {
+      game();
+
+      return;
+    }
+
+    const result = bullsAndCows(secretNumber, guess);
+
+    !result.bulls === 4 && game();
```

Comment:
This code isn't readable
It's the statement for checking if you should call `game` one more time, right? Let's write it with if / else statement